jetty9-base/src/main/jetty-base/etc/java-util-logging.properties

```diff
@@ -0,0 +1,9 @@
+.level=INFO
+
+handlers=com.google.cloud.logging.LoggingHandler
+com.google.cloud.logging.LoggingHandler.level=FINE
+com.google.cloud.logging.LoggingHandler.log=gae_app.log
```

Comment:
Maybe this log name should be `jetty.log`? This would especially make sense if someone configures a different logging framework for their app like log4j. On the other hand users relying on the default config may find it strange to see their app logs in `jetty.log` stream. WDYT?